tools/proofers/for_mentors.php

```diff
@@ -181,16 +187,26 @@ function output_project_details($mentored_round, $projectid, $nameofwork, $autho
 
 // -------------------------------------------------------------------
 
-function get_beginner_projects_in_state($state)
+function get_beginner_projects_in_state($state, $mentored_round)
 {
+    $mentored_round_detail = $mentored_round . '.proj_done';
+
     $sql = sprintf("
-        SELECT projectid, nameofwork, authorsname
-        FROM projects
+        SELECT
+            projects.projectid,
+            projects.nameofwork,
+            projects.authorsname,
+            MAX(project_events.timestamp) AS t_left_mentee_round
+        FROM projects LEFT OUTER JOIN project_events USING (projectid)
         WHERE
-            difficulty = 'BEGINNER' AND
-            state='%s'
+            difficulty = 'BEGINNER'
+            AND project_events.event_type = 'transition'
+            AND project_events.details2 = '%s'
+            AND projects.state='%s'
+        GROUP BY projects.projectid
         ORDER BY
-            modifieddate ASC",
+            MAX(project_events.timestamp) ASC",
```

Comment:
I've never seen a function in an order by like this so I don't know what it does. Is it necessary?

Reply:
Speaking of the `MAX` being part of it? I think I tried just ordering by `project_events.timestamp`, and mysql didn't like it. I'll go back and double-check from the mysql command line, though. Maybe it was something else that was the problem at the time, and not that.

Reply:
Hm. Didn't even need to go to the command line -- just took it out in the code, and the code still works, so it must've been something else when I was testing the query.